index.js

```diff
@@ -71,7 +71,15 @@ function redisStore(args) {
       }
 
       if (opts.parse) {
-        result = JSON.parse(result);
+
+        try {
+          // allow undefined only if allowed by isCacheableValue
+          if(! ( (result === undefined || result === "undefined") && typeof args.isCacheableValue === 'function' && args.isCacheableValue(result))) {
```

Comment:
Ok I understand, this is used both in get and set. I was thinking it was only a get since there is some JSON.parsing.

Why do we parse it in a set?